spec/features/purchases_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.feature 'Purchases', type: :feature do
+  before (:all) do
+    FactoryBot.create(:user, name: 'Ellen')
+    FactoryBot.create(:user, name: 'Ben')
+    FactoryBot.create(:user, name: 'Both')
+    FactoryBot.create(:subcategory)
+    FactoryBot.create_list(:purchase, 28)
+  end
+
+  before (:each) do
+    visit root_path
+  end
+
+  scenario 'display twenty purchases by default' do
+    click_link 'Purchases'
+
+    numrows = page.all(:css, 'table tr').size - 1 # subtracting header row
```

Comment:
Tests look good - you could probably use the selector `table tbody tr`, so that you don't pick up the header row